stuff/profiles/layouts/settings/xsheet.css

```diff
@@ -1,5 +1,5 @@
 .header {
-  border-type: inser;
+  border-type: inset;
   border: 1px solid black;
```

Comment:
Didn't test to see if this worked. I just looked up the CSS standard and found that `inser` was invalid.